lib/leadership.js

```diff
@@ -82,16 +122,8 @@ BasicLeadershipElection.prototype.check = function () {
 	}
 };
 
-BasicLeadershipElection.prototype.start = function (discover, callback) {
-	this.discover = discover;
```

Comment:
Per other review comment, I believe this should be kept.

Reply:
Understood, if you are inheriting from this, then that makes sense to leave it. It is not necessary though for the internal default use of the module.